### PR TITLE
[cssom-view-1] Add dictionary default values

### PR DESCRIPTION
**Pull request by saschanaz**
Required after heycam/webidl#750. (Found from web-platform-tests/wpt#18382)

### CODE REVIEW

**Comment by saschanaz** [comment]
Marked as non substantive for IPR from ash-nazg.

File: cssom-view-1/Overview.bs [hunk 1]
```diff
@@ -1072,12 +1072,12 @@ dictionary ScrollIntoViewOptions : ScrollOptions {
 partial interface Element {
   DOMRectList getClientRects();
   [NewObject] DOMRect getBoundingClientRect();
-  void scrollIntoView(optional (boolean or ScrollIntoViewOptions) arg);
-  void scroll(optional ScrollToOptions options);
+  void scrollIntoView(optional (boolean or ScrollIntoViewOptions) arg = {});
```
Comment (dbaron) [inline cssom-view-1/Overview.bs]: Are you sure this is the right thing given the `(boolean or ScrollIntoViewOptions)`?

Comment (saschanaz) [reply cssom-view-1/Overview.bs]: Yes. Previously the default value was implicitly an empty dictionary when the union includes non-required dictionary type, and now it must be explicit.

Comment (saschanaz) [reply cssom-view-1/Overview.bs]: Check out the relevant spec change: https://github.com/heycam/webidl/pull/750/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR2245

**Pull request merged**